legal-api/src/legal_api/models/review.py

```diff
@@ -76,6 +76,27 @@ def get_review(cls, filing_id) -> Review:
                       one_or_none())
         return review
 
+    @classmethod
+    def get_paginated_reviews(cls, page, limit):
+        """Return paginated reviews."""
+        query = db.session.query(Review).order_by(Review.creation_date.asc())
```

Comment:
👍 Yes, the default sort is oldest first. This is probably worth a comment, but then again, this code will be expanded soon to handle filtering and sorting by different fields.